source/mainnet/variables.rst

```diff
@@ -13,8 +13,8 @@
 .. |bw| replace:: Concordium Wallet for Web
 
 .. Client verification variables
-.. |client-linux| replace:: concordium-client_5.0.2-0
-.. |client-version| replace:: 5.0.2-0
+.. |client-linux| replace:: concordium-client_5.1.0-0
+.. |client-version| replace:: 5.1.0-0
 .. |client-linux-checksum| replace:: bd39487463b45c7bbbbcdeeb1cbbd0f4fe8e62041869aa3db9cdcf331ed8a049
```

Comment:
Yes. That should be done after building the release binary, or am I mistaken?

Reply:
Do we know the version number yet? I can see you're reviewing something older because I replaced 0.2 with x.x in the draft pull request.

Reply:
Kasper means the version number and the checksum. Of course, we have to wait for the build to get the checksum. :-)

Reply:
I bumped the minor version of the client, i.e. from 5.0.2 to 5.1.0, and AFAIK that's what's being released.

Reply:
So Martin is testing it, right? If so, I will wait for the version since sometimes bugs need to be fixed. We have a week to prepare this, so I'm not worried.

Reply:
You don't need to do anything with it. I will handle it before the release.

Reply:
I see! Thanks. And yes, Martin is testing it.

Reply:
Yes, I was referring to the checksum (the line, which the comment was attached to :p).